### PR TITLE
View/Delete/Replay single dead job

### PR DESCRIPTION
**Pull request by alishamohanty**
View, delete, replay a [single dead job](http://139.59.73.117:3000/redis/dead).
image

### CODE REVIEW

File: src/goose/brokers/redis/console/pages/dead.clj [hunk 1]
```diff
@@ -77,6 +79,27 @@
...
+  [:div.redis
+   [:h1 "Dead Job"]
+   (if job
+     [:div
+      [:form {:action (str base-path "/job/" id)
+              :method "post"}
+       [:div
+        (c/action-btns [(c/replay-btn {:disabled false})
+                        (c/delete-btn
+                          "Are you sure you want to the delete job?"
```
Comment (siripr4) [inline src/goose/brokers/redis/console/pages/dead.clj:94]:
```suggestion
                          "Are you sure you want to delete the job?"
```

File: src/goose/brokers/redis/console/pages/dead.clj [hunk 2]
```diff
@@ -77,6 +79,27 @@
...
+                          {:disabled false})])
+        [:input {:name  "job"
+                 :type  "hidden"
+                 :value (utils/encode-to-str job)}]
+        (c/job-table job)]]]
+     (c/flash-msg {:type    :error
+                   :message "No job found"}))])
```
Comment (siripr4) [inline src/goose/brokers/redis/console/pages/dead.clj:101]:
```suggestion
                   :message "No jobs found"}))])
```

**Review by siripr4 — APPROVED** [review]

File: src/goose/brokers/redis/console/pages/dead.clj (hunk 2 above)
Comment (alishamohanty) [reply src/goose/brokers/redis/console/pages/dead.clj:101]: @siripr4 since this is single job's view page I think, no job sounds better.

Comment (siripr4) [reply src/goose/brokers/redis/console/pages/dead.clj:101]: `Job not found`, then? Either is fine actually
And I'm assuming it's too late to change this

**Pull request merged**